src/app/pages/StoryPage/index.jsx

```diff
@@ -95,8 +95,16 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
     }
   `;
 
-  const StyledGrid = styled(Grid)`
+  const storyPageGelMaxWidths = css`
```

Comment:
Is there a reason why this CSS was chosen to be stored in a constant? It seems to be only used in `StyledGrid` so is worth having a constant for it apart from context?

Reply:
you're right @HarveyPeachey, I'll move it within `StyledGrid`